app/builders/v2/report_builder.rb

```diff
@@ -19,7 +19,7 @@ def timeseries
   # For backward compatible with old report
   def build
     timeseries.each_with_object([]) do |p, arr|
-      arr << { value: p[1], timestamp: p[0].to_time.to_i }
+      arr << { value: p[1], timestamp: p[0].in_time_zone(@timezone).to_i }
```

Comment:
Is there any chance this update will change/break existing behavior downstream? It seems like the intended result, but I don't have 100% context on how this is used.

Reply:
It should not change/break the existing behavior. @pranavrajs any thoughts?

Reply:
@sojan-official Please take a lead on this.